libsql-server/src/namespace/meta_store.rs

```diff
@@ -409,6 +447,22 @@ impl MetaStoreHandle {
         }
     }
 
+    pub async fn link(&self, shared_schema: NamespaceName, linked_db: NamespaceName) -> Result<()> {
+        match &self.inner {
+            HandleState::Internal(_) => {
+                //FIXME: what is this even for?
```

Comment:
Question for @MarinPostma or @LucioFranco : what do we use `HandleState::Internal` for? I seen it used only for some tests, but it's dubious way to have hardcoded execution path that bypasses actual application logic for the sake of test.